doc/INSTALL.md

```diff
@@ -42,3 +42,7 @@ civicrm_api3('DoctorWhen', 'run', array(
   'tasks' => '*',
 ));
 ```
+
+## Uninstall
+
+While DoctorWhen is an extremely useful extension for databases that have been upgraded over time, it has no ongoing functionality at this time.  Leaving the extension installed could potentially create problems later.  Therefore, it is recommended that DoctorWhen be uninstalled after confirming that it has successfully done it's job.
```

Comment:
1. `INSTALL.md` mostly shows specific technical steps for installation/usage. For uninstalling, I think the specific are steps are more like (a) `cv ext:uninstall doctorwhen` and/or (b) navigate to "Administer => System Settings => Extensions" and disable/uninstall and/or (c)  `rm -rf /path/to/doctorwhen`. (But written in a prettier/clearer way than what I just typed.)

2. On the merits/details of the text above, I'm not certain what problems could arise -- though I agree with the general advice. FWIW, I look at it this way:

    * Keeping `doctorwhen` active could be *confusing* to some admin users -- because it puts something in the GUI that they don't need on a day-to-day basis. For people providing support, disabling `doctorwhen` can head-off potential questions or prevent unnecessary usage.
    * After the transforms have been run, I expect the potential for damage from re-running it goes way down. It's probably not zero, but I personally don't know any specific dangers.
    * You *may* need to use DoctorWhen again in the future -- if new temporal issues are identified which need cleanup.
    * However, there's really no guarantee that the cleanup-logic will be the same for those future issues. If/when there are more issues, you should get the latest version. There's not much value in keeping the old version.

That's probably a bit too much information -- I like how your other PR's sort of side-step the rationale and keep the messaging simple. But if we

Reply:
Makes sense.  If this were the dev doc, it might be more appropriate.  I'll see what I can come up with for uninstall info, but I like the idea of keeping it super simple.  As an aside, I like the layout you are laying down for this and it seems like a good template for other extensions.  That said, should something like an Uninstall section point towards official docs and otherwise only point out specific deviations or warnings?

Reply:
As to your second point, as expected I'm not seeing it show up on the available extensions list, which means that it won't be upgraded without manual intervention.  An admin who inherits the system with the existing install has the potential to do damage to an upgraded database that a newer version of the extension that matched the updated code base would circumvent.  Yes, it's unlikely as the tests and checks are reasonable today.  In the end, we are saying the same thing, but I'm more apt to put more pressure on expecting a disaster.  It's the lack of automated upgrade that gives me more pause.  In the end we get to the same point, keep it simple and tell them to uninstall.